src/main/res/layout/export_progress_activity.xml

```diff
@@ -0,0 +1,141 @@
+            app:layout_constraintBottom_toBottomOf="@id/export_progress_slash"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/export_progress_slash"
+            app:layout_constraintTop_toTopOf="@id/export_progress_slash"
+            tools:text="10" />
+
+        <ProgressBar
+            android:id="@+id/export_progress_bar"
+            style="@android:style/Widget.ProgressBar.Horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="24dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginRight="24dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/export_progress_done"
+            tools:progress="25" />
+
+        <TextView
+            android:id="@+id/export_progress_summary"
+            style="@style/TextMedium"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="24dp"
+            android:text="@string/export_progress_review"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/export_progress_bar" />
+
+        <ImageView
+            android:id="@+id/export_progress_alert_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="24dp"
+            android:layout_marginEnd="8dp"
+            app:layout_constraintHorizontal_chainStyle="packed"
+            app:layout_constraintBottom_toBottomOf="@id/export_progress_alert_msg"
+            app:layout_constraintEnd_toStartOf="@id/export_progress_alert_msg"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@id/export_progress_alert_msg"
+            app:tint="@android:color/holo_red_dark"
+            app:srcCompat="@drawable/ic_report_problem_24" />
+
+        <TextView
+            android:id="@+id/export_progress_alert_msg"
+            style="@style/TextMedium"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="24dp"
+            android:layout_marginBottom="24dp"
+            app:layout_constraintBottom_toTopOf="@id/export_progress_toggle"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/export_progress_alert_icon"
+            app:layout_constraintTop_toBottomOf="@id/export_progress_summary"
+            app:layout_constraintVertical_chainStyle="packed"
+            tools:text="The track already exists" />
+
+        <androidx.appcompat.widget.SwitchCompat
```

Comment:
Why not use a checkbox?
https://material.io/develop/android/components/checkboxes